components/collections/src/codepointinvliststringlist/mod.rs

```diff
@@ -228,37 +228,7 @@ impl<'a> FromIterator<&'a str> for CodePointInversionListAndStringList<'_> {
         // Ensure that the string list is sorted. If not, the binary search that
         // is used for `.contains(&str)` will return garbase otuput.
         strings.sort_unstable();
-
-        // Ensure that `strings` is deduplicated. Check for any duplicates first.
-        let is_deduped = strings.windows(2).all(|adjacents| {
-            if let (Some(s1), Some(s2)) = (adjacents.get(0), adjacents.get(1)) {
-                s1 != s2
-            } else {
-                false
-            }
-        });
-        let strings = if is_deduped {
-            strings
-        } else {
-            let mut deduped_strs = Vec::<&str>::new();
-            let mut strs_iter = strings.iter();
-
-            let first_str = strs_iter.next();
-            debug_assert!(first_str.is_some()); // If `strings` was empty or len() < 2, then
-                                                // `is_deduped_strs` would have been true.
-            if let Some(mut next_str) = first_str {
-                deduped_strs.push(*next_str);
-
-                for s in strs_iter {
-                    if s != next_str {
-                        deduped_strs.push(*s);
-                        next_str = s;
-                    }
-                }
-            }
-
-            deduped_strs
-        };
+        strings.dedup();
```

Comment:
@echeran can you sanity check that it is ok to use stdlib method instead of manual code here?

Reply:
There's a `dedup` in stdlib? I thought it was only in itertools; this is great

Reply:
I'm guessing that this is the [`Vec::dedup()`](https://doc.rust-lang.org/stable/std/vec/struct.Vec.html#method.dedup) method.  Has the same behavior, and the previous line still exists to sort it first to prepare for the intended result, so it looks good.